Backpack-SwiftUI/ImageGallerySlideshow/Classes/BPKImageGallerySlideshow.swift

```diff
@@ -44,15 +44,15 @@ struct ImageGallerySlideshow<ImageView: View>: ViewModifier {
         @Binding var currentIndex: Int
         
         var body: some View {
-            GeometryReader { proxy in
-                VStack(spacing: .xl) {
-                    ImageGalleryHeader(
-                        closeAccessibilityLabel: closeAccessibilityLabel,
-                        onCloseTapped: onCloseTapped
-                    )
-                    .padding([.leading, .top], .base)
-                    
-                    ZStack(alignment: .bottom) {
+            VStack(spacing: .xl) {
+                ImageGalleryHeader(
+                    closeAccessibilityLabel: closeAccessibilityLabel,
+                    onCloseTapped: onCloseTapped
+                )
+                .padding([.leading, .top], .base)
+                
+                ZStack(alignment: .bottom) {
+                    GeometryReader { proxy in
```

Comment:
The only change we made here is move the `GeometryReader` from line `47` to `55`. 
![image](https://github.com/user-attachments/assets/6ea03dce-a737-4411-87c3-0f49df884f46)

| Before | After |
| -- | -- |
| ![image](https://github.com/user-attachments/assets/692a9226-ee7a-472a-85ab-a244edfb1dac) | ![image](https://github.com/user-attachments/assets/41629fba-f92a-4676-9443-c9ae54cf825c) |

Reply:
Interesting investigation!!